packages/data-table/src/_stories/DataTable.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { Meta, StoryObj } from '@storybook/react'
+import CdcDataTable from '../CdcDataTable'
+
+import DataTableConfig from '../../examples/data-table-example.json'
+import { editConfigKeys } from '../helpers/configHelpers'
+
+const meta: Meta<typeof CdcDataTable> = {
+  title: 'Components/Templates/DataTable',
+  component: CdcDataTable
+}
+
+type Story = StoryObj<typeof CdcDataTable>
+
+export const DataTable: Story = {
+  args: {
+    config: editConfigKeys(DataTableConfig, [{ path: ['filters'], value: [] }]),
```

Comment:
this is much more readable to me:

{ ...DataTableConfig, filters: [] }

is that not what you're trying to do here?